app/tests/base.py

```diff
@@ -0,0 +1,61 @@
+from rest_framework.test import APIClient, APITestCase
+
+from ..models import (LanguageProficiency, Pronoun, Role, Skill,
+                      SkillProficiency, SpokenLanguage, User)
+
+
+class BaseTestCase(APITestCase):
+
+    def setUp(self):
+        self.client = APIClient()
+
+        self.role = Role.objects.create(
```

Comment:
Just a preference, but this doesn't have to be multiline

Reply:
okay